docs/source/format/Versioning.rst

```diff
@@ -18,12 +18,13 @@
 Format Versioning and Stability
 ===============================
 
-Starting with version 1.0.0, Apache Arrow utilizes
+Starting with version 1.0.0, Apache Arrow uses
 **two versions** to describe each release of the project:
 the **Format Version** and the **Library Version**. Each Library
 Version has a corresponding Format Version, and multiple versions of
 the library may have the same format version. For example, library
-versions 2.0.0 and 3.0.0 may both track format version 1.0.0.
+versions 2.0.0 and 3.0.0 may both track format version 1.0.0. See
+:ref:`status` for details about supported features in each library.
```

Comment:
The `ref` directive points to a logical anchor, but here I think you want to point to the document named "status":
```suggestion
:doc:`status` for details about supported features in each library.
```